src/components/auth/SignUpScreen.js

```diff
@@ -1,290 +1,303 @@
-                      onSubmitEditing={(event) => { this.refs.SecondInput._root.focus() }}
-                      onChangeText={value => this.onChangeText('name', value)}
-                    />
-                  </Item>
-                  {/* Email */}
-                  <Item style={styles.itemStyle}>
-                    <Ionicons style={styles.iconStyle1} name="ios-mail" />
-                    <Input
-                      style={styles.input}
-                      placeholder={i18n.t('Email')}
-                      placeholderTextColor={COLORS.lightblue}
-                      returnKeyType='next'
-                      autoCapitalize='none'
-                      autoCorrect={false}
-                      keyboardType={'email-address'}
-                      ref='SecondInput'
-                      onSubmitEditing={(event) => { this.refs.ThirdInput._root.focus() }}
-                      onChangeText={value => this.onChangeText('email', value)}
-                    />
-                  </Item>
-                  {/* Password */}
-                  <Item style={styles.itemStyle}>
-                    <Ionicons style={styles.iconStyle1} name="ios-lock" />
-                    <Input
-                      style={styles.input}
-                      placeholder={i18n.t('Password')}
-                      placeholderTextColor={COLORS.lightblue}
-                      returnKeyType='next'
-                      autoCapitalize='none'
-                      autoCorrect={false}
-                      secureTextEntry={this.state.hidePassword1}
-                      ref='ThirdInput'
-                      onSubmitEditing={(event) => { this.refs.FourthInput._root.focus() }}
-                      onChangeText={value => this.onChangeText('password', value)}
-                    />
-                    <Ionicons style={styles.iconStyle2} name="ios-eye" onPress={() => this.handleHidePassword1()}/>
-                  </Item>
-                  {/* Confirm Password */}
-                  <Item style={styles.itemStyle}>
-                    <Ionicons style={styles.iconStyle1} name="ios-lock" />
-                    <Input
-                      style={styles.input}
-                      placeholder={i18n.t('Confirm Password')}
-                      placeholderTextColor={COLORS.lightblue}
-                      returnKeyType='go'
-                      autoCapitalize='none'
-                      autoCorrect={false}
-                      secureTextEntry={this.state.hidePassword2}
-                      ref='FourthInput'
-                      onChangeText={value => this.onChangeText('password_confirmation', value)}
+              <Text style={styles.header}>Olá!</Text>
+              <Text style={styles.paragraph}>
+                Excepteur commodo deserunt eu ad labore labore qui ullamco
+                cillum consectetur incididunt eiusmod Lorem. Sunt do ipsum id
+                officia officia.
+              </Text>
```

Comment:
I think slightly smaller text can be used in general but also scaled
![image](https://user-images.githubusercontent.com/5898380/86957717-f9edc180-c10f-11ea-9870-7fe44c701139.png)

Reply:
I'll make the font a bit smaller at baseline. This is just template text from a lorem ipsum generator so it's not really representative of translations or text we may actually work with.